### DISCUSSION TITLE
Neptune r6g instance sizes

### DISCUSSION THREAD

**Discussion by aaronatbissell** [aws/aws-connected-device-framework#75] [Ideas]
Has anyone done any testing around using Neptune r6g instance types?

**Comment by jonasneu-aws** [comment]
Hi @aaronatbissell, I saw [the announcement](https://aws.amazon.com/about-aws/whats-new/2021/11/aws-graviton2-based-instances-amazon-neptune/) for the availability of these instance types for Neptune, but I am not aware of anyone running CDF asset library with the r6g instance type. Potential performance gains aside, is there anything in particular you'd want to test for?

**Reply by aaronatbissell** [reply]
Nothing specific - I just didn't know if there were any obvious "gotcha's" or reasons to avoid it. I'm really interested in the performance and cost benefits with these new instance types, and I didn't know if someone had already made the switch and saw gains similar to the reports in the announcement.
